pandas/core/groupby.py

```diff
@@ -929,6 +929,10 @@ def mean(self):
         Compute mean of groups, excluding missing values
 
         For multiple groupings, the result index will be a MultiIndex
+
+        Do not call this function via the numpy library. numpy's mean
```

Comment:
How is what I'm saying encouraging it?  It explicitly says "Do not call," and I'm not sure I understand how not saying anything about it works.  It seems like people already had that idea without us even saying anything about it.  Perhaps there needs to be one central place where it warns users about the dangers of calling these functions via the `numpy` library.  `scipy` does that for example with their `sparse` matrix implementation.